web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -315,11 +360,13 @@ const TeamReviews = ({ onBack, periodId }) => {
       if (!reviewee) return [];
       const { id } = reviewee;
       const as = assignments.filter(a => a.revieweeId === id) ?? [];
-      const reviewerIds = new Set();
```

Comment:
This code is inside the `getReviewers` function. I changed to logic to add an `approved` property to the member objects. This is used in the code below that renders the reviewer `Chip` components so it knows what background color to use and whether the row should end with an APPROVE button or an UNAPPROVE button.